src/components/Chat.js

```diff
@@ -1,3 +1,7 @@
+//useRef, io, link, edit, make_room, chat_bubble, and rocket are defined but never used
```

Comment:
`link, edit, make_room, chat_bubble,` and `rocket` don't seem to appear in this file at all. Were they already removed?

Reply:
It seems like it is still in the current commit and may be used for future implementation so I will leave it